### ISSUE TITLE
Front-end highlighter redesign: New window links missing warning and decorative image incorrectly announced

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1617]
Example code:
```
<a class="edac-highlight-panel-description-reference" href="https://a11ychecker.com/help1969?utm_source=accessibility-checker&amp;utm_medium=software&amp;utm_campaign=frontend-highlighter&amp;php_version=8.3.30&amp;platform=wordpress&amp;platform_version=6.9.4&amp;software=free&amp;software_version=1.39.0&amp;days_active=0&amp;utm_content=possible_heading" target="_blank" rel="noopener noreferrer">How to Fix ↗︎</a>

<a class="edac-highlight-panel-description-reference" href="https://a11ychecker.com/help1969?utm_source=accessibility-checker&amp;utm_medium=software&amp;utm_campaign=frontend-highlighter&amp;php_version=8.3.30&amp;platform=wordpress&amp;platform_version=6.9.4&amp;software=free&amp;software_version=1.39.0&amp;days_active=0&amp;utm_content=possible_heading" target="_blank" rel="noopener noreferrer">1.3.1 Info and Relationships ↗︎</a>
```

Example screenshot with VoiceOver:

<img width="1048" height="458" alt="Image" src="https://github.com/user-attachments/assets/54e54c1c-bb5b-4038-8888-e4df13f4f2af" />

Voiceover says "Link, how to fix northeast arrow"

How to fix:

1. Wrap all new window arrows in a span with `aria-hidden="true"`
2. Add translatable screen reader text "opens a new window" or similar.
